spread.yaml

```diff
@@ -119,7 +119,7 @@ prepare: |
   sudo snap wait system seed.loaded
   # IDK why but if we don't sleep after this we get:
   # error: cannot install "snapd": Post "https://api.snapcraft.io/v2/snaps/refresh": context canceled
-  sleep 2
+  sleep 4
```

Comment:
It may be worth to check with snapd people if there's a better way to ensure we're ready instead of relying on random sleeps.

Reply:
Definitely agree - I meant to bring this up with @zyga at the last snapcraft office hours, but there was actual snapcraft-related stuff to talk about.